### DISCUSSION TITLE
Xresources: wm.program.launcher.* not really working

### DISCUSSION THREAD

**Discussion by dampfwal3477** [regolith-linux/regolith-desktop#999] [General]
Hey,
I tried to modify the default program launchers in my ~/.config/regolith3/Xresources file to e.g. launch rofi instead of Ilia with Meta+Space. Sadly rofi gave me an error about a unknown -t flag. Realized I had to remove it within  /usr/share/regolith/i3/config.d/20_ilia which didn't seem to be the right way to do this.
Any better ideas for that? 
Thanks in advance.

**Comment by kgilmer** [comment]
Hi @dampfwal3477 , yeah I think you can just uninstall the debian package that provides the unneeded config partial.  That can be determinted w/ `dpkg`.  Example from my computer:

```shell
$ dpkg -S 20_ilia
regolith-sway-ilia: /usr/share/regolith/sway/config.d/20_ilia
regolith-i3-ilia: /usr/share/regolith/i3/config.d/20_ilia
```

Since you're running `i3` you could uninstall `regolith-i3-ilia` but take a look at what `apt` will actually do before proceeding.  In my case:

```shell
$ sudo apt remove regolith-i3-ilia
...
The following packages will be REMOVED:
  regolith-i3-ilia
...
```

If you try to remove a package that other packages depend upon, `apt` will tell you the whole set of packages that would be removed.  For Regolith's config partials, they've been designed for this use-case of removal, and as such are "soft" dependencies and should not break your desktop session if removed.
